apps/demos/Demos/DataGrid/AIColumns/description.md

```diff
@@ -0,0 +1,12 @@
+DevExtreme DataGrid supports AI columns. These columns augment your data and transform our DataGrid into an AI-powered data exploration tool. This demo implements an AI column (fixed on the right side of the component) to display one additional AI-generated data field. You can also integrate multiple AI columns to expand the scope of generated insights.
+
+<!--split-->
+
+To integrate an AI column, do the following:
+
+- Configure [aiIntegration](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#aiIntegration) (or **columns[]**.**ai**.[aiIntegration](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/ai/#aiIntegration)).
+- Set a column's [type](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#type) to *"ai"*.
+- Specify the column's [name](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#name).
+- Configure **columns[]**.**ai** options, including the [prompt](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/ai/#prompt) and generation [mode](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/ai/#mode).
+
+This demo also configures the [onAIColumnRequestCreating](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onAIColumnRequestCreating) handler to limit data included in AI requests. The default DataGrid behavior is to include all data from visible rows in AI requests, including data not bound to a column and data of hidden columns. This gives LLMs broader context, but increases the component's use of AI resources. To limit data included in AI requests, modify the **AIColumnRequestCreatingEvent**.[data](/Documentation/ApiReference/UI_Components/dxDataGrid/Types/AIColumnRequestCreatingEvent/#data) parameter.
```

Comment:
AFAIK we have some limitations for AI service that we use in this demo. 
So, it's better to include some Note about it to warning users that the demo is for testing purposes and we've limited tokens...
Please check similar notes in other DX /not-DX products and consult with developer to finalize the text

Reply:
In Blazor, we actually have two notes (see the example "DevExpress AI-powered extensions follow the "bring your own key" principle": https://blazor/serverside-25.2/AI/SemanticSearch#Content ). Please check if the second note is also applicable to your demo.

If yes, you can use them as-is, since they have been already checked a couple of times.